src/registry/SynchronousRegistry.ts

```diff
@@ -64,6 +64,9 @@ export abstract class SynchronousRegistry extends BaseRegistry implements IRegis
     return this.getWarpRoutesForIds([routeId])[0] || null;
   }
 
+  /**
+   * Retrieves a filtered map of the warp routes artifacts
```

Comment:
Let's try to be consistent with our nomenclature. I don't think I call these artifacts anywhere in the registry code but I could be wrong. The problem with 'artifact' is that while it is the output of one tool, it's also the input to another.